### PR TITLE
docs: Add contributing guidelines

### LINKED ISSUE #48: Add Contributing Guideline
We should add Contributing Guideline

### PR DESCRIPTION
**Pull request by thepranaygupta**
fixes #48

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+
+### Alternatively, Git CLI way to Sync forked repository with origin repository:
+```bash
+git fetch upstream
+```
+```bash
+git merge upstream/main
+```
+### [Github Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for Syncing
```
Comment (suitupstan) [inline CONTRIBUTING.md:49]: Why is this a heading?

**Review by suitupstan — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (thepranaygupta) [reply CONTRIBUTING.md:49]: I followed the format of [LinkFree](https://github.com/EddieHubCommunity/LinkFree), should I change it to normal text?

Comment (suitupstan) [reply CONTRIBUTING.md:49]: Ah I see, if it were added there as a heading it must be for some reason 🤔

**Review by suitupstan — APPROVED** [review]

**Review by EmmaDawsonDev — APPROVED** [review]
LGTM!

**Review by Pradumnasaraf — APPROVED** [review]
LGTM 🐰

**Pull request merged**